lib/packwerk/constant_fixture_inspector.rb

```diff
@@ -0,0 +1,60 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "packwerk/constant_name_inspector"
+require "packwerk/constant_fixture_inspector/fixture_retriever"
+require "packwerk/node"
+
+module Packwerk
+  # Exracts the constant reference from a fixture call
+  class ConstantFixtureInspector
```

Comment:
I don't know what a `ConstantFixture` is, so I'd rather call this `FixtureReferenceInspector`